src/AppRouter.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import Hello from './hello';
+import Hello from './Hello';
```

Comment:
Nice catch with the name capitalization. Like @larsbrekken mentioned, we'll need to change the name of the file as well now from `src/hello.js` to `src/Hello.js`